src/components/calcite-color-picker/calcite-color-picker.scss

```diff
@@ -84,95 +82,94 @@ $gap--large: 12px;
 
 :host([appearance="minimal"]) {
   .container {
-    border: none;
+    @apply border-none;
   }
 }
 
 .container {
-  display: inline-block;
-  border: 1px solid var(--calcite-ui-border-1);
-  background-color: var(--calcite-ui-foreground-1);
+  @apply inline-block
+  border
+  border-solid
+  border-color-1
+  bg-foreground-1;
 }
 
 .color-field-and-slider {
   margin-bottom: -16px;
 
   &--interactive {
-    cursor: pointer;
+    @apply cursor-pointer;
   }
 }
 
 .control-section {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  @apply flex flex-row flex-wrap;
 }
 
 .section {
-  padding: 0 $section-padding $section-padding;
+  @apply pt-0 px-3 pb-3;
 
   &:first-of-type {
-    padding-top: $section-padding;
+    @apply pt-3;
   }
 }
 
 .color-hex-options,
 .section--split {
-  flex-grow: 1;
+  @apply flex-grow;
 }
 
 .header {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  color: var(--calcite-ui-text-3);
-  font-weight: 500;
-  @include font-size(-2);
+  @apply flex
+    items-center
+    justify-between
+    text-color-3
+    font-medium
+    text--2-wrap;
 
   &.header--underlined {
-    border-bottom: 1px solid var(--calcite-ui-border-1);
+    @apply border-b
+      border-solid
+      border-color-1;
   }
 }
 
 .header--hex {
-  line-height: 1.5;
-  @include font-size(-2);
-  padding: $section-padding 0 15px;
+  @apply leading-normal
+    text--2h
+    pt-3 px-0 pb-4;
 }
 
 .control {
-  margin-top: 8px;
+  @apply pt-2;
 }
 
 .channels {
-  display: flex;
-  justify-content: space-between;
+  @apply flex justify-between;
 }
 
 .channel {
+  //FIXME: Maybe this can be 1/3?
   width: 31%;
 }
 
 .saved-colors {
-  padding-top: $section-padding;
-  display: -ms-grid;
-  display: grid;
+  @apply pt-3
+  grid
+  gap-2
+  w-full;
   grid-template-columns: repeat(auto-fill, minmax(24px, 1fr));
-  grid-gap: $gap;
-  width: 100%;
 }
 
 .saved-colors-buttons {
-  display: flex;
+  @apply flex;
 }
 
 .saved-color {
-  @include focus-style-base();
-
-  cursor: pointer;
+  @apply focus-base cursor-pointer;
 
   &:focus {
-    @include focus-style-outset();
+    @apply focus-outset;
   }
 
   &:hover {
```

Comment:
Missed?

Reply:
actually, I didn't know how to approach this.

Reply:
Lets add this to the list